install/disconnected_install.adoc

```diff
@@ -540,91 +548,76 @@ $ systemctl start httpd
 enabled=1
 gpgcheck=0
 [rhel-7-server-ose-3.10-rpms]
 name=rhel-7-server-ose-3.10-rpms
-baseurl=http://<server_IP>/repos/rhel-7-server-ose-3.10-rpms
+baseurl=http://<server_IP>/repos/rhel-7-server-ose-3.10-rpms <1>
 enabled=1
 gpgcheck=0
 ----
+<1> Replace `<server_IP>` with the IP address or host name of the Apache server
+that hosts the software repositories.
 
-[[disconnected-host-preparations]]
-=== Host Preparation
-
-At this point, the systems are ready to continue to be prepared
-xref:host_preparation.adoc#install-config-install-host-preparation[following
-the {product-title} documentation].
-
-Skip the section titled *Host Registration* and start with *Installing Base Packages*.
-
-[[disconnected-installing-openshift]]
-== Installing {product-title}
-
-[[disconnected-importing-containerized-components]]
-=== Importing {product-title} Component Images
-
-To import the relevant components, securely copy the images from the connected
-host to the individual {product-title} hosts:
-
+. To import the relevant software and images, securely copy the images from the
+webserver to the individual {product-title} hosts:
++
 [source, bash]
 ----
 $ scp /var/www/html/repos/images/ose3-images.tar root@<openshift_host_name>:
 $ ssh root@<openshift_host_name> "docker load -i ose3-images.tar"
 $ scp /var/www/html/images/ose3-builder-images.tar root@<openshift_master_host_name>:
 $ ssh root@<openshift_master_host_name> "docker load -i ose3-builder-images.tar"
+$ ssh root@<openshift_master_host_name> "ose3-logging-metrics-images.tar"
+$ ssh root@<openshift_master_host_name> "ose3-builder-images.tar"
 ----
 
-[[disconnected-running-the-openshift-installer]]
-=== Running the {product-title} Installer
+. Finish preparing the hosts for installation. Follow the
+xref:host_preparation.adoc#install-config-install-host-preparation[Preparing your hosts]
+steps, omitting the steps in the *Host Registration* section.
 
-You can now follow the standard installation documentation to:
+[[disconnected-installing-openshift]]
+== Installing {product-title}
 
-. xref:configuring_inventory_file.adoc#configuring-ansible[Configure your
-inventory file] and
-. xref:running_install.adoc#install-running-installation-playbooks[Run the
+After you prepare the hosts in your disconnected environment:
+* xref:configuring_inventory_file.adoc#configuring-ansible[Configure your
+inventory file].
+* xref:running_install.adoc#install-running-installation-playbooks[Run the
 installation playbooks].
 
 [IMPORTANT]
 ====
 For
-xref:index.adoc#planning-installation-types[system container installations], to install the etcd container, you can set the Ansible variable `osm_etcd_image` to be the fully qualified name of the etcd image on
+xref:index.adoc#planning-installation-types[system container installations] on
+RHEL Atomic Host, to install the etcd container, you can set the Ansible variable
+`osm_etcd_image` to be the fully qualified name of the etcd image on
```

Comment:
I do not think osm_etcd_image is only specific for install on RHEL Atomic Host, now etcd is running a static pod, osm_etcd_image should be valid to any install.

Reply:
Yes, that's correct, there's nothing unique to Atomic Host about osm_etcd_image. Also, is this not evaluated from oreg_url like other images?

Reply:
This is the only reference to osm_etcd_image that I see in the entire collection. If the oreg_url takes care of it, please let me know if I can remove this note.

Reply:
When osm_etcd_image is not defined, oreg_url would take care of it, I still suggest to keep the reference to osm_etcd_image, because several customers are using the var in their disconnected install, but need polish the sentence.

Reply:
I'll retain the original text. Thank you!